Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -164,5 +178,20 @@ public void updateMotorTelemetry() {
         leftDriveSlave.updateTelemetryProperties();
         rightDrive.updateTelemetryProperties();
         rightDriveSlave.updateTelemetryProperties();
+        
+        leftDistanceProp.set(leftDrive.getPosition());
+        rightDistanceProp.set(rightDrive.getPosition());
+    }
+    
+    public double getDistance() {
+        return convertToInches(leftDrive.getPosition() - startPosition.get());
+    }
+    
+    public void resetDistance() {
+        startPosition.set(convertToInches(leftDrive.getPosition()));
```

Comment:
This seems odd; we're using a property to frequently store data for immediate access by other code. I think this value should be stored in a local variable independent of the property.